### PR TITLE
2.0.0.Final announcement

### PR DESCRIPTION
**Pull request by Naros**

### CODE REVIEW

File: _posts/2022-10-17-debezium-2-0-final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+
+If you are already using the transaction metadata feature, new events will contain this field after upgrading.
+
+If you are not using the transaction metadata feature but find this useful, simply add the `provide.transaction.metadata` option set to _true_ to your connector configuration.
+By default, metadata events are emitted to a topic named after your `database.server.name` option.
+This can be overridden by specifying the `transaction.topic` option, as shown below:
+
+[source,properties]
+----
+database.server.name=server1
```
Comment (jpechane) [inline _posts/2022-10-17-debezium-2-0-final-released.adoc]: AFAIK this is no longer true as the topic naming has changed later on.

File: _posts/2022-10-17-debezium-2-0-final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+
+In Debezium 1.8, we introduced the new MongoDB change stream feature while also deprecating the oplog implementation.
+The transition to change streams offers a variety of benefits, such as being able to stream changes from non-primary nodes,
+the ability to emit update events with a full document representation for downstream consumers,
+and so much more.  In short, change streams is just a much more superior way to perform change data capture with MongoDB.
+
+The removal of the oplog implementation also means that MongoDB 3.x is no longer supported.
+If you are using MongoDB 3.x, you will need to upgrade to at least MongoDB 4.0 or later with Debezium 2.0.
+
+=== Before state support (MongoDb 6.0)
```
Comment (jpechane) [inline _posts/2022-10-17-debezium-2-0-final-released.adoc]:
```suggestion
=== Before state support (MongoDB 6.0)
```

**Comment by jpechane** [comment]
@Naros Nice work, thanks! I've just left two comments. When you address them please feel free to merge the PR and make the announcement.

**Comment by jpechane** [comment]
@Naros Also tutorial seems to be updated to use 2.0 config options but the docs seems to render `1.9` tags in the image names. Can it be caused by having two stable versions?

**Comment by Naros** [comment]
> Can it be caused by having two stable versions?

Nope, this appears to be because the `debezium-docker-label` inside of the `antora.yml` file still refers to `1.9` rather than `2.0`.  I'll submit a fix to the main repository and get this sorted out ASAP.

**Pull request merged**